<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RundownSeparatorWidget</class>
 <widget class="QWidget" name="RundownSeparatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>36</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameItem">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <widget class="QLabel" name="labelIcon">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>2</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelLabel">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>2</y>
        <width>531</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelColor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroupColor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>6</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>labelColor</zorder>
     <zorder>labelIcon</zorder>
     <zorder>labelLabel</zorder>
     <zorder>labelGroupColor</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameStatus">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>8</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLabel" name="labelActiveColor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>8</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>8</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>8</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>addToCombo()</slot>
 </slots>
</ui>
